### PR TITLE
Pioneer 13a asset management initial push

### PR DESCRIPTION
**Pull request by cdobs**
We are deploying 3 gliders today (340 D10, 380 D8, 388D9) on cruise TS20191212

Changes made:
-Added TS20191212 to cruise info.csv with approximate return time, this will need to be updated once the cruise is finished
-Updated deployment csvs for 340, 380 and 388 for new deployments
-Also updated 340 deployment csv with recovery info from previous cruise
-2 out of the 3 gliders have freshly calibrated sensors (340 does not)
-Confirmed that all sensors are present in bulkload (DVLs with xducer sns as well)
-Added the following calibration CSVs for sensors in 380/388:

New CTDGVM cals:
-00073 20180822
-09059 20180109

New FLORTM cals:
-SN3185 20190724
-SN4471 20181101

New DOSTAM cals:
-SN 123 20191009
-SN 323 20190812

New PARADM cals:
-SN 50151 20190813
-SN 50244 20190813

### CODE REVIEW

File: calibration/DOSTAM/CGINS-DOSTAM-00123__20191009.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+123,CC_calibration_date,1539043200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00123__20191009.csv]: I think the date format in the vendor cal doc is day-month-yr so this should be 20190910 (updated vendor filename in Vault); milliseconds needs to be updated as well (make sure you do yr 2019...what you have currently is for 2018 plus the wrong day-month)

File: calibration/DOSTAM/CGINS-DOSTAM-00323__20190812.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+323,CC_calibration_date,1565568000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00323__20190812.csv]: looks good

File: calibration/FLORTM/CGINS-FLORTM-03185__20190724.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3185,CC_angular_resolution,1.076,
+3185,CC_depolarization_ratio,0.039,
+3185,CC_measurement_wavelength,700,
+3185,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-03185__20190724.csv]: looks good

File: calibration/FLORTM/CGINS-FLORTM-04471__20181101.csv [hunk 4]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4471,CC_angular_resolution,1.076,
+4471,CC_depolarization_ratio,0.039,
+4471,CC_measurement_wavelength,700,
+4471,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04471__20181101.csv]: looks good

File: calibration/PARADM/CGINS-PARADM-50151__20190813.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50151,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50151__20190813.csv]: looks good

File: calibration/PARADM/CGINS-PARADM-50244__20190813.csv [hunk 6]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50244,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50244__20190813.csv]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-09059__20180109.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9059,CC_calibration_date,1515456000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09059__20180109.csv]: looks good - are we putting file_source info in (same question for all)?

File: calibration/CTDGVM/CGINS-CTDGVM-00073__20190822.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+73,CC_calibration_date,1566432000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-00073__20190822.csv]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00123__20190910.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+123,CC_calibration_date,1568073600000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00123__20190910.csv]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-09059__20180109.csv (hunk 7 above)
Comment (pbrickley) [reply calibration/CTDGVM/CGINS-CTDGVM-09059__20180109.csv]: Looks ok, but I have the same question.

File: calibration/CTDGVM/CGINS-CTDGVM-00073__20180822.csv [hunk 10]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+73,CC_calibration_date,1534896000000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/CTDGVM/CGINS-CTDGVM-00073__20180822.csv]: looks ok

File: calibration/DOSTAM/CGINS-DOSTAM-00123__20190910.csv (hunk 9 above)
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00123__20190910.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00323__20190812.csv (hunk 2 above)
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00323__20190812.csv]: ok

File: calibration/FLORTM/CGINS-FLORTM-03185__20190724.csv (hunk 3 above)
Comment (pbrickley) [reply calibration/FLORTM/CGINS-FLORTM-03185__20190724.csv]: ok

File: calibration/FLORTM/CGINS-FLORTM-04471__20181101.csv (hunk 4 above)
Comment (pbrickley) [reply calibration/FLORTM/CGINS-FLORTM-04471__20181101.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50151__20190813.csv (hunk 5 above)
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50151__20190813.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50244__20190813.csv (hunk 6 above)
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50244__20190813.csv]: ok

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 11]
```diff
@@ -40,4 +40,10 @@ AR-31,,AR-31,,CP05MOAS-GL380-01-ADCPAM000,7,1,2018-10-25T20:23:00,2018-10-27T02:
...
+AR-31,,AR-31,,CP05MOAS-GL380-05-PARADM000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50277,39.87505,-70.88473,,1000,1000,SS-1
+TS20191212,,,,CP05MOAS-GL380-00-ENG000000,8,1,2019-12-12T22:31:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.91493,-70.66606,,1000,1000,FZ-1
+TS20191212,,,,CP05MOAS-GL380-01-ADCPAM000,8,1,2019-12-12T22:31:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-49697,39.91493,-70.66606,,1000,1000,FZ-1
+TS20191212,,,,CP05MOAS-GL380-02-FLORTM000,8,1,2019-12-12T22:31:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04471,39.91493,-70.66606,,1000,1000,FZ-1
+TS20191212,,,,CP05MOAS-GL380-03-CTDGVM000,8,1,2019-12-12T22:31:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09059,39.91493,-70.66606,,1000,1000,FZ-1
+TS20191212,,,,CP05MOAS-GL380-04-DOSTAM000,8,1,2019-12-12T22:31:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00123,39.91493,-70.66606,,1000,1000,FZ-1
+TS20191212,,,,CP05MOAS-GL380-05-PARADM000,8,1,2019-12-12T22:31:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50151,39.91493,-70.66606,,1000,1000,FZ-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL380_Deploy.csv]: deployment date/time look good

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 12]
```diff
@@ -47,3 +47,9 @@ WJ20190617,,AR-38,,CP05MOAS-GL388-02-FLORTM000,8,1,2019-06-18T22:58:15,2019-09-2
...
 WJ20190617,,AR-38,,CP05MOAS-GL388-05-PARADM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50203,39.92227,-70.91603,,1000,1000,SS-2
+TS20191212,,,,CP05MOAS-GL388-00-ENG000000,9,1,2019-12-12T22:47:17,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.914233,-70.665167,,1000,1000,SS-1
+TS20191212,,,,CP05MOAS-GL388-01-ADCPAM000,9,1,2019-12-12T22:47:17,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-49016,39.914233,-70.665167,,1000,1000,SS-1
+TS20191212,,,,CP05MOAS-GL388-02-FLORTM000,9,1,2019-12-12T22:47:17,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03185,39.914233,-70.665167,,1000,1000,SS-1
+TS20191212,,,,CP05MOAS-GL388-03-CTDGVM000,9,1,2019-12-12T22:47:17,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-00073,39.914233,-70.665167,,1000,1000,SS-1
+TS20191212,,,,CP05MOAS-GL388-04-DOSTAM000,9,1,2019-12-12T22:47:17,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00323,39.914233,-70.665167,,1000,1000,SS-1
+TS20191212,,,,CP05MOAS-GL388-05-PARADM000,9,1,2019-12-12T22:47:17,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50244,39.914233,-70.665167,,1000,1000,SS-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL388_Deploy.csv]: deployment date/time look good.

File: deployment/CP05MOAS-GL340_Deploy.csv [hunk 13]
```diff
@@ -47,9 +47,15 @@ TS20190205,,BK20190504,,CP05MOAS-GL340-03-CTDGVM000,8,1,2019-02-05T08:24:00,2019
...
+AR-39,,AR-40,,CP05MOAS-GL340-01-ADCPAM000,9,1,2019-09-27T12:26:00,2019-11-05T15:00:21,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0501,-70.6002,,200,200,EB 
+TS20191212,,,,CP05MOAS-GL340-00-ENG000000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,40.0817,-70.74813,,200,200,EB 
+TS20191212,,,,CP05MOAS-GL340-02-FLORTM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-03921,40.0817,-70.74813,,200,200,EB 
+TS20191212,,,,CP05MOAS-GL340-03-CTDGVM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09029,40.0817,-70.74813,,200,200,EB 
+TS20191212,,,,CP05MOAS-GL340-04-DOSTAM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,40.0817,-70.74813,,200,200,EB 
+TS20191212,,,,CP05MOAS-GL340-05-PARADM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,40.0817,-70.74813,,200,200,EB 
+TS20191212,,,,CP05MOAS-GL340-01-ADCPAM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0817,-70.74813,,200,200,EB
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL340_Deploy.csv]: deployment date/time look good

File: cruise/CruiseInformation.csv [hunk 14]
```diff
@@ -183,3 +183,4 @@ AR-39,R/V Neil Armstrong,2019-09-25T16:55:00,2019-10-15T16:26:00,Pioneer 13
 SKQ201921S,R/V Sikuliaq,2019-10-10T05:00:00,2019-10-24T00:00:00,Endurance 12
 AR-40,R/V Neil Armstrong,2019-10-20T12:00:00,2019-11-06T16:00:00,Pioneer glider cp_340 recovery
 EK20191121,R/V Elakha,2019-11-21T16:14:00,2019-11-22T00:08:00,Endurance CSPP and glider operations
+TS20191212,F/V Tenacious II,2019-12-12T15:00:00,2019-12-13T23:59:00,Pioneer 13a glider deployment
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply cruise/CruiseInformation.csv]: ok, but return time is probably a guess

**Pull request merged**